doc/nrf/config_and_build/sysbuild_migration.rst

```diff
@@ -0,0 +1,579 @@
++---------------------------------------------------------------------------+--------------------------------------------------------------------------------------------------------------------------+
+| :kconfig:option:`SB_CONFIG_MCUBOOT_HW_DOWNGRADE_PREVENTION_COUNTER_SLOTS` | Number of available hardware counter slots for downgrade prevention                                                      |
++---------------------------------------------------------------------------+--------------------------------------------------------------------------------------------------------------------------+
+| :kconfig:option:`SB_CONFIG_MCUBOOT_HW_DOWNGRADE_PREVENTION_COUNTER_VALUE` | Security counter value of the image for downgrade prevention                                                             |
++---------------------------------------------------------------------------+--------------------------------------------------------------------------------------------------------------------------+
+| :kconfig:option:`SB_CONFIG_MCUBOOT_UPDATEABLE_IMAGES`                     | Number of updateable images for MCUboot to support                                                                       |
++---------------------------------------------------------------------------+--------------------------------------------------------------------------------------------------------------------------+
+| :kconfig:option:`SB_CONFIG_MCUBOOT_APP_SYNC_UPDATEABLE_IMAGES`            | Will set the main application number of updateable images as well as MCUboot if enabled, otherwise will only set MCUboot |
++---------------------------------------------------------------------------+--------------------------------------------------------------------------------------------------------------------------+
+| :kconfig:option:`SB_CONFIG_SECURE_BOOT_MCUBOOT_VERSION`                   | MCUboot version string to use when creating MCUboot update package for application secure boot mode                      |
++---------------------------------------------------------------------------+--------------------------------------------------------------------------------------------------------------------------+
+
+Support for unsigned images has been added, as well as image encryption, these options will generate the respective output files for the main application build. Any MCUboot configuration that was previously done in the main application or MCUboot needs to be updated to apply it at a sysbuild level, if this is not done then settings of these builds will be forcefully replaced with the default which sysbuild generates, which will make firmware updates incompatible with firmware images built in previous versions of |NCS|.
+
+.. _child_parent_to_sysbuild_migration_secure_boot:
+
+Secure boot
+###########
+
+Secure boot mode selection and key file configuration is handled by sysbuild, the following Kconfig options exist:
```

Comment:
Sysbuild handles the mode selection of secure boot and the configuration of the key file .
The following Kconfig options are available: